src/org/zaproxy/zap/extension/fuzz/ZapAddOn.xml

```diff
@@ -7,9 +7,7 @@
     <author>ZAP Dev Team</author>
     <url>https://github.com/zaproxy/zap-core-help/wiki/HelpAddonsFuzzConcepts</url>
     <changes>
-    <![CDATA[
-    Fix modification of file and file fuzzers in Windows.<br>
```

Comment:
Old change should be kept, the add-on was not yet released with this fix (otherwise the version would have to be bumped as well).